gcc/rust/expand/rust-expand-visitor.cc

```diff
@@ -730,12 +835,15 @@ ExpandVisitor::visit (AST::ClosureExprInner &expr)
 void
 ExpandVisitor::visit (AST::BlockExpr &expr)
 {
-  expand_inner_stmts (expr.get_statements ());
+  expand_inner_stmts (expr, expr.get_statements ());
```

Comment:
```suggestion
  expand_inner_stmts (expr);
```

Reply:
Fixed in e145c1eb94d76af6dee7f1ec0bc5f7c92f1d2779